File: ui/visual-model-component/src/main/java/org/visual/model/ui/widget/DebugWidget.java
package org.visual.model.ui.widget;

import javafx.scene.layout.StackPane;

public class DebugWidget extends StackPane {

    public DebugWidget() {
        setLayoutX(10.0);
        setLayoutY(10.0);
        setPrefSize(50, 50);
        FpsWidget fpsWidget = new FpsWidget();
        getChildren().add(fpsWidget);
        toFront();
    }
}

File: visual-model-debugger/src/main/kotlin/org/visual/model/debugger/component/DebugWidget.kt
package org.visual.model.debugger.component

import javafx.scene.layout.StackPane
import org.visual.model.component.widget.FpsWidget

class DebugWidget : StackPane() {
  init {
    layoutX = 10.0
    layoutY = 10.0
    setPrefSize(50.0, 50.0)
    val fpsWidget = FpsWidget()
    children.add(fpsWidget)
    toFront()
  }
}
